### PR TITLE
Added documentation.

### PR DESCRIPTION
**Pull request by delrueba**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# CONTRIBUTING
+
+Firstly, this library was created to go hand in hand with the [city of Ghent styleguide](https://stijlgids.stad.gent),
```
Comment (zero2one) [inline .github/CONTRIBUTING.md]: Text lines within a markdown file should not be longer than 80 characters.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+1. Document your changes inline in the code using [JSDoc 3](http://usejsdoc.org/). Update the README.md with details of changes to the interface.
+2. Write tests if you add any new functionality. 
+2. Increase the version numbers in the `package.json` to the new version that this Pull Request would represent. The versioning scheme we use is [SemVer](https://semver.org/).
+3. You may merge the Pull Request in once you have the sign-off of at least one other developer, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## Release a new version
+
+### Prerequisites
+
+* Have [git flow](https://github.com/nvie/gitflow) installed on your machine
```
Comment (zero2one) [inline .github/CONTRIBUTING.md]: Sentences should end with a period.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.0.1] - 2020-03-12
```
Comment (zero2one) [inline CHANGELOG.md]: Add the actual link [0.0.1] to the end of this file.
Add the [Unreleased] link to the file

E.g.

```markdown
[0.0.1]: https://github.com/StadGent/npm_package_modal/releases/tag/0.0.1
[Unreleased]: https://github.com/StadGent/npm_package_modal/compare/master...develop
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+# City of Ghent Modal
+
+This is a fully accessible modal library. 
+It includes a body scroll lock, tab trap, route events and aria-hidden for obscured contents.
```
Comment (zero2one) [inline README.md]: Text lines should not be longer than 80 chars.

**Review by zero2one — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (delrueba) [reply CHANGELOG.md]: Haven't actually released anything yet. I'll remove it.

**Pull request merged**